.github/workflows/e2e.yaml

```diff
@@ -29,18 +28,22 @@ jobs:
       id: short-sha
       with:
         length: 10
-    - run: sudo apt-get install fuse-overlayfs
```

Comment:
Did you mean to remove this?

Reply:
Yes, it is [already installed](https://github.com/openshift-psap/special-resource-operator/runs/4153675034?check_suite_focus=true#step:5:10).